Dockerfile.composer

```diff
@@ -0,0 +1,13 @@
+FROM composer:1.6.4
+
+ARG composer_dev_arg
+
+COPY composer.json \
+    composer.lock \
+    ./
+
+RUN composer --no-interaction install ${composer_dev_arg} --ignore-platform-reqs --no-autoloader --no-scripts --no-suggest
+
+COPY test/ test/
```

Comment:
There is a `src/` here, but it doesn't have any classes; it could still be put in for simplicity

Reply:
Things are only here for the autoloader. Not having it saves superfluous cache busting (and future classes not appearing in the autoloader will be an obvious fail).

Reply:
Ideally `test` wouldn't be copied if `${composer_dev_arg}` is empty.